src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -161,12 +161,14 @@ const useCloudEnvironmentPolling = namespace => {
   }
   const load = async () => {
     try {
-      const [newDisks, newClusters] = await Promise.all([
+      const [newDisks, newClusters, hiddenDisks] = await Promise.all([
         Ajax(signal).Disks.list({ googleProject: namespace, creator: getUser().email }),
-        Ajax(signal).Clusters.list({ googleProject: namespace, creator: getUser().email })
+        Ajax(signal).Clusters.list({ googleProject: namespace, creator: getUser().email }),
+        Ajax(signal).Disks.list({ googleProject: namespace, creator: getUser().email, hidden: 'true' })
```

Comment:
TOL: We need an extra call here because the list endpoint doesn't return labels, right? It might be good to highlight this to the Leo team, and see about including that data so we don't have to make multiple calls.

Reply:
They actually are aware of this! We were talking about it and this is the solution I ended up brainstorming. I can flag it again though!

Reply:
Yeah, they have pretty reasonable reasons for not having it return labels: https://broadinstitute.slack.com/archives/CA3NP1733/p1601302225045000